.bumpversion.cfg

```diff
@@ -0,0 +1,21 @@
+[bumpversion]
```

Comment:
What does this file do?

Reply:
It's config for [bumpversion](https://github.com/peritus/bumpversion) tool.
It defines how version numbers are progressing and use can use the tool to release a new version: it will update relevant places and create a git commit and tag.

Here it is: 1.4.0.dev0 ->  1.4.0.dev1 -> ... -> 1.4.0rc0 -> 1.4.0rc1 -> ... -> 1.4.0.
I don't like `rc0` so usually I manually override it.